client/questions_answers/helpers.js

```diff
@@ -45,4 +45,25 @@ const sortAnswersList = (unsortedList) => {
   return sellerResponses.concat(sortedList);
 };
 
-export { formatAnswererName, formatDate, sortAnswersList };
\ No newline at end of file
+const validateEmail = (email) => {
+  const re = /^[^\s@]+@[^\s@]+$/;
+  return re.test(email);
+};
+
+const validateFormFields = (question, nickname, email) => {
+
+  let problem = '';
+  if (!question) {
+    problem += '\n question';
+  }
+  if (!nickname) {
+    problem += '\n nickname';
+  }
+  if (!validateEmail(email)) {
+    problem += '\n email';
+  }
+  return problem ? `Please enter a valid: ${problem}` : null;
```

Comment:
Curious if the identified 'problem' needs to be in a new line, per the requirements (assuming the \n means that string will be in a new line below 'Please enter a valid'). If not, why not leave it on the same line and remove the colon? Just to make it a bit more readable. But maybe I'm missing something spelled out in the requirements!

Reply:
Totally. It's set up so that if there is a problem with the question, nickname, and/or email each problem will appear on a new line below "Please enter a valid: ". I re-read the docs and it should actually read: "You must enter the following:" so I gotta make that one little change. There might be a better way to format this, as you suggest--and I would love your input on that. Gonna merge now but if you have time could you take a look at the alert pop-ups and give me some suggestions after you've merged the new master branch? That would be awesome.